sfm_pc/management/commands/import_google_doc.py

```diff
@@ -226,16 +195,10 @@ def handle(self, *args, **options):
         # Disconnect post save signals
         self.disconnectSignals()
 
-        if options.get('location_doc_id'):
-            self.get_locations_from_drive(options['location_doc_id'])
-            self.create_locations()
-
-        if options.get('folder'):
-            self.stdout.write('Loading data from folder {}...'.format(options['folder']))
-            all_sheets = self.get_sheets_from_folder(options['folder'])
-        else:
-            self.stdout.write('Loading data from Google Sheets...')
-            all_sheets = self.get_sheets_from_doc(options['doc_id'], options['source_doc_id'])
+        if (country_directory := options.get('country_directory').strip()):
```

Comment:
Good question, I was just following the existing pattern. It checked if the variables existed to determine if it would load via google docs or local files. So now that is pointless.